.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```diff
@@ -159,7 +159,7 @@ stages:
             displayName: "Get AKS Cluster"
           - script: |
               kubectl delete ns load-test
-              cilium connectivity test --connect-timeout 4s --request-timeout 30s
+              cilium connectivity test --connect-timeout 4s --request-timeout 30s --test '!pod-to-pod-encryption,!node-to-node-encryption'

```

Comment:
Can we add a comment specifying from which cli version this is added from.
And what cilium agent it supports. We need to keep a track of it once we are on higher versions of cilium i guess.

Reply:
I can add a comment `Enable these tests when encryption feature is enabled` , but the reason why the tests were originally disabled was because we have encryption disabled. Upstream Cilium CLI update changed it to where only if the cluster is a single node cluster do these tests not run. IMO as long as we have encryption disabled we should not run these tests and explicitly defining it as such will ensure future changes to this test do not impact us.